### PR TITLE
feat: added landing page

### PR DESCRIPTION
**Pull request by pradipgharge**
Created the Landing page for Furnbuy-store

Screenshot 2022-05-04 at 1 00 07 PM

### CODE REVIEW

**Review by sri-rishi — APPROVED** [review]
Code looks good, feel free to merge

**Comment by abhisheksahani-nova** [comment]
Looks clean and subtle, good to merge

**Review by abhisheksahani-nova — APPROVED** [review]

File: src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          <h4 className="footer-column-heading">Categories</h4>
+          <ul className="footer-list">
+            <li className="footer-list-item">Chairs & Sofas</li>
+            <li className="footer-list-item">Lamp & Lightings</li>
+            <li className="footer-list-item">Storage & Shelves</li>
+            <li className="footer-list-item">Living Room</li>
+          </ul>
+        </div>
+        <div className="footer-column">
+          <h4 className="footer-col-heading">Useful Links</h4>
```
Comment (anik31) [inline src/components/Footer/Footer.jsx:25]: suggestion - you can store this data in array of objects and map over it. It will look clean.

**Review by anik31 — APPROVED** [review]
Everything looks great, code is clean & UI is very nice

**Pull request merged**